Lib/test/test_except_star.py

```diff
@@ -1000,5 +1000,192 @@ def test_exc_info_restored(self):
         self.assertEqual(sys.exc_info(), (None, None, None))
 
 
+class TestExceptStar_WeirdLeafExceptions(ExceptStarTest):
+    # Test that except* works when leaf exceptions are
+    # unhashable or have a bad custom __eq__
+
+    class UnhashableExc(ValueError):
+        hash = None
```

Comment:
Shouldn't this be `__hash__` instead of `hash`?

Reply:
Yes of course, thanks!